### ISSUE TITLE
mintUpdate lists errors messages as packages to upgrade when refreshing the packages list fails

### ISSUE THREAD

**Issue by glebihan** [linuxmint/mintupdate#4]
While one of the repository servers wasn't responding and mintUpdate could not refresh the list of packages, I clicked on "Install Updates" anyway.
This caused mintUpdate to list 434 new packages to be installed, including strings which came from errors messages, eg :
 * MarkUpgrade()
 * a
 * called
 * non-upgradable
 * ...

I wasn't crazy enough to let it go ahead with the upgrade and see what it would do :)

**Comment by xenopeek** [comment]
I can't reproduce this on mintupdate 5.1.0. Perhaps this can be closed.

I tried to reproduce it by faking a non-responding server by adding a non-routable address to sources.list with the line: `deb http://10.255.255.1 xenial main`. Update Manager (after a long wait) responds with a dialog:
> W: Failed to fetch http://10.255.255.1/dists/xenial/InRelease  Could not connect to 10.255.255.1:80 (10.255.255.1), connection timed out
> W: Some index files failed to download. They have been ignored, or old ones used instead.

There's only a Close button on the dialog so I pressed that. Installing updates immediately after that did not reproduce the issue described.
